### ISSUE TITLE
Squids not Spawning

### ISSUE THREAD

**Issue by ShadowGeneral** [cybercat5555/BetterAnimals#15]
Not sure if this is part of the mod or something else, but as soon as I start to wander around the world squids stop spawning.

If this is not part of the mod, then could you tell me where I should go to find out how to fix this.

**Comment by itsmeow** [comment]
No. This mod cannot cause that. It is impossible for a client only mod to stop squids from spawning.

**Comment by Dabombber** [comment]
While this mod shouldn't affect the server, I've found using it in combination with betteranimalsplus and crafttweaker does stop squids from spawning in singleplayer. I'm not sure which mod is to blame, it's most likely some weird edge case since any two of them work fine.

A possible workaround would be to run a dedicated server to properly separate the client and server code, but that may use more RAM than singleplayer.
A quick, dumb and ugly fix which seems to work is renaming the better animals mod to zbetteranimals-5.2.1.jar so it's loaded after crafttweaker.

Tested with the following:
- minecraft 1.12.2
- forge 1.12.2-14.23.5.2847
- betteranimals-5.2.1
- betteranimalsplus-1.12.2-7.1.1
- CraftTweaker2-1.12-4.1.19

 Although it might seem to work at first, that's just because squids (sometimes) still spawn during chunk generation. The easiest way to see the problem is using the command "/kill @e[type=squid]" over an ocean (or [type=!player] just to be sure). Afterwards "/testfor @e[type=squid]" will show no entities, where they should respawn almost instantly.
